subcommands/status/cmd.go

```diff
@@ -38,29 +38,66 @@ func showStatus(cmd *cobra.Command, args []string) {
 	subcommands.DieNotNil(err)
 
 	fmt.Println("Total number of devices:", status.TotalDevices)
-	fmt.Println("")
+
+	if len(status.ProdTags) > 0 || len(status.ProdWaveTags) > 0 {
+		fmt.Println("\nProduction devices:")
+		t := tabby.New()
+		t.AddHeader("TAG", "LATEST TARGET", "DEVICES", "ON LATEST", "ON ORPHAN", "ONLINE")
+		for idx, tag := range append(status.ProdWaveTags, status.ProdTags...) {
+			name := tag.Name
+			if idx < len(status.ProdWaveTags) {
+				// Wave devices are always tagged
+				name += " (wave)"
+			} else if len(name) == 0 {
+				// Prod devices can be untagged, although they cannot fetch updates
+				name = "(Untagged)"
+			}
+			t.AddLine(name, tag.LatestTarget, tag.DevicesTotal, tag.DevicesOnLatest,
+				tag.DevicesOnOrphan, tag.DevicesOnline)
+		}
+		t.Print()
+
+		fmt.Println("\nTest devices:")
+	}
+
 	t := tabby.New()
 	t.AddHeader("TAG", "LATEST TARGET", "DEVICES", "ON LATEST", "ONLINE")
 	for _, tag := range status.Tags {
-		if len(tag.Name) == 0 {
-			// These are untagged devices:
-			tag.Name = "(Untagged)"
+		name := tag.Name
+		if len(name) == 0 {
+			// Test devices can be untagged and can even fetch updates
+			name = "(Untagged)"
 		}
-		t.AddLine(tag.Name, tag.LatestTarget, tag.DevicesTotal, tag.DevicesOnLatest, tag.DevicesOnline)
+		t.AddLine(name, tag.LatestTarget, tag.DevicesTotal, tag.DevicesOnLatest, tag.DevicesOnline)
 	}
 	t.Print()
 
-	for _, tag := range status.Tags {
-		if len(tag.Name) == 0 {
+	fmt.Println("\nOrphan target versions below are marked with a star (*)")
+	printTargetStatus("Active Prod Wave", status.ProdWaveTags)
+	printTargetStatus("Prod", status.ProdTags)
```

Comment:
Let's use the full word "Production" for label.

Reply:
ok